src/actions/constants.js

```diff
@@ -1,2 +1,4 @@
 export const INCREMENT_COUNTER = 'INCREMENT_COUNTER'
 export const DECREMENT_COUNTER = 'DECREMENT_COUNTER'
+
+export const TOGGLER = 'TOGGLER'
```

Comment:
This should describe an action - should be in the verb form. i.e. "TOGGLE," or "TOGGLE_TEXT".